spec.html

```diff
@@ -7192,7 +7168,7 @@ <h1>GetBindingValue ( _N_, _S_ )</h1>
 
         <emu-clause id="sec-module-environment-records-deletebinding-n">
           <h1>DeleteBinding ( _N_ )</h1>
-          <p>The concrete Environment Record method DeleteBinding for module Environment Records refuses to delete bindings.</p>
+          <p>The DeleteBinding concrete method of a module Environment Record _envRec_ takes argument _N_ (a String). It performs the following steps when called:</p>
```

Comment:
Personally I would prefer that we not use the standard preamble for methods which are never invoked. I think it reads better as

> The DeleteBinding concrete method of a module Environment Record is never used within this specification.

and

> The CreateImmutableBinding concrete method of an object Environment Record is never used within this specification.

though I am fine with keeping the algorithm steps which assert the same thing, I guess.

Reply:
I go back and forth about how these two should be handled. For now, I'm going to use your suggested wording and drop the `<emu-alg>`s.